src/model/mapping/mapping_model.py

```
@@ -395,38 +436,40 @@ def _map_nomenclatures_chunk(
     try:
-        noms['group'] = get_nomenclatures_groups(
+        noms['internal_group'] = get_nomenclatures_groups(
             noms=noms,
             model_id=model_id,
-            is_use_params=is_use_params,
+            # is_use_params=is_use_params,
         )
     except ValueError:
         raise Exception(
             f"Model with ID '{model_id}' does not support params or DataFrame for prediction does not contains them"
         )
```

Ruff

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)